### DISCUSSION TITLE
Update Documentation for V1

### DISCUSSION THREAD

**Discussion by lewoudar** [vitalik/django-ninja#931] [General]
Hi everyone,
First of all, thanks to all the contributors who contributed to V1. This is an important step for the project :)
However, it seems like the documentation is not updated, and we all know the importance of documentation for an open-source project. It would be nice to have it updated.

Kind regards

**Comment by vitalik** [comment]
@lewoudar which exactly documentation piece you see not updated ?

**Comment by lewoudar** [comment]
Hi @vitalik , my bad, I think I looked at the documentation too quickly. I was looking at the [Query Parameters](https://django-ninja.dev/guides/input/query-params/) and I didn't see any reference to the new way to declare arguments. But at the end, there is clearly a [section](https://django-ninja.dev/guides/input/query-params/#using-schema) with the new annotation. Sorry for the noise.
